flips/20201125-bip-44-multi-account.md

```diff
@@ -9,188 +9,279 @@
-be able to reduce friction for users who wish to create accounts on Flow.
+By following this standard, developers of existing BIP 44 crypto wallets will be
+able to reduce friction for users who wish to create accounts on Flow.
+
+By standardizing BIP 44 usage across wallets, users should be able to use their
+BIP 39 mnemonic codes across wallets to manage access to the same keys and accounts.
 
 ## Design Proposal
 
 ### Path Levels
 
-BIP 44 defines an implementation of 
-[Bitcoin Improvement Proposal 32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) (BIP 32)
-that standardizes a format for hierarchical deterministic (HD) 
-key derivation paths that support multiple coins and multiple accounts per coin.
+BIP 44 defines an implementation of [Bitcoin Improvement Proposal
+32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) (BIP 32)
+that standardizes a format for hierarchical deterministic (HD) key derivation
+paths that support multiple coins and multiple accounts per coin.
 
 The BIP 44 path format is as follows:
 
 ```
 m / purpose' / coin_type' / account' / change / address_index
 ```
 
-When using this format to construct or discover a Flow user wallet, 
-the value(s) of each path parameter should follow the specifications below.
+When using this format to construct or discover a Flow user wallet, the value(s)
+of each path parameter should follow the specifications below.
 
 #### Purpose
 
-Purpose is always the constant 44' (0x8000002C), 
-as described in [BIP 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#purpose).
+Purpose is always the constant 44' (0x8000002C), as described in [BIP
+44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#purpose).
 
 #### Coin Type
 
-Coin type is always the constant 539' (0x8000021b), 
-the unique identifier for the Flow protocol, as defined in 
-[SatoshiLabs Improvement Proposal 44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) (SLIP 44).
+Coin type is always the constant 539' (0x8000021b), the unique identifier for
+the Flow protocol, as defined in [SatoshiLabs Improvement Proposal
+44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) (SLIP 44).
 
 #### Account
 
-Account is the index of a distinct account address owned by the wallet user. 
-No two account indices should be used to generate child public keys that are assigned 
-to the same Flow account address. Furthermore, a single account index should not
-be used to generate keys for more than one account address.
+Account is the index of a distinct account address owned by the wallet user. No
+two account indices should be used to generate child public keys that are
+assigned to the same Flow account address. Furthermore, a single account index
+should not be used to generate keys for more than one account address.
+
+The Account Index to use when generating a key pair to be added on a new account
```

Comment:
```suggestion
The account index to use when generating a key pair to be added on a new account
```

Personally I'd like to avoid capitalizing terms that aren't proper nouns. I find it difficult to read and don't think it adds much clarity.